lib/oauth/request_proxy/action_controller_request.rb

```diff
@@ -67,7 +67,7 @@ def parameters_for_signature
 
       params.
         join('&').split('&').
-        reject(&:blank?).
+        reject { |s| s.match(/\A\s*\z/) }.
```

Comment:
actually `empty?` would be enough if there is no chance for `nil` => https://github.com/rails/rails/blob/42b0c942520e59399d70c2170253aa5275a42af1/activesupport/lib/active_support/core_ext/object/blank.rb#L16-L18